### DISCUSSION TITLE
non riesco a caricare ais-catcher

### DISCUSSION THREAD

**Discussion by kewinxx** [jvde-github/AIS-catcher#131] [Q&A]
salve  perdonatemi il disturbo  ma non riesco a scaricare il file ais-catcher

**Comment by jvde-github** [comment]
don't speak Italian, sorry but which file?

**Reply by kewinxx** [reply]
ais-catcher 4.9 no dowunload

**Reply by kewinxx** [reply]
trovato ok grazie mille e scusi il disturbo

**Reply by jvde-github** [reply]
No problem! Have fun with it.

**Comment by jvde-github** [comment]
seeems to be solved so closing discussion.
